src/program/types/ClassBody.js

```diff
@@ -49,6 +49,15 @@ export default class ClassBody extends Node {
 				if (!inFunctionExpression) code.appendLeft(constructor.end, ';');
 			}
 
+			const classFields = [];
+			this.body.forEach(method => {
```

Comment:
I think that argument name is misleading here :) 
```suggestion
			this.body.forEach(element => {
```